### PR TITLE
Cleanup legacy stuff

### PR DESCRIPTION
**Pull request by alexsporn**
Removed unused plugins and packages

### CODE REVIEW

**Review by piotrm50 — APPROVED** [review]
LGTM, but let's wait for some second review :smile:

**Review by muXxer — APPROVED** [review]
:trollface:

**Review by jonastheis — APPROVED** [review]

**Pull request merged**
